include/swift/Remote/MetadataReader.h

```diff
@@ -524,6 +525,34 @@ class MetadataReader {
       return None;
 
     return start;
+#else
+    // All swift class instances start with an isa pointer,
```

Comment:
@mikeash I found this info in a presentation you gave in 2016 and your blog. Is this still accurate? (By the way, your blog is great!)

Reply:
I think just about everything I talked about is still correct. (Various things have changed with metadata and such, but I didn't get into that.) The particular details you mention here are correct.

Reply:
Oh, and thanks, glad you found it useful!